tests/test_providers/test_business.py

```diff
@@ -28,7 +28,11 @@ def test_copyright(self, business):
         assert '©' in result
 
     def test_currency_iso_code(self, _business):
```

Comment:
So, you need to use business fixture here to cover all locales.

Reply:
@ceccoemi You can find such fixture [here](https://github.com/lk-geimfari/mimesis/blob/c3ea6ae31659bbba5502939ca3130b0c63fca744/tests/conftest.py#L26)

Reply:
@lk-geimfari I fixed the fixture! Please see my last commit.